src/styles/index.scss

```diff
@@ -1,3 +1,70 @@
-body {
+@keyframes rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.page {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  min-width: 100vw;
+}
+
+.stopwatch {
+  position: relative;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  height: 80vmin;
+  width: 80vmin;
+
+  &__element {
+    height: 50%;
+    width: 100%;
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+
+    &--circle {
+      height: 100%;
+      align-items: center;
+    }
+  }
+
+  &__hand-minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    animation: rotate 3600s steps(60) infinite;
+    transform-origin: bottom;
```

Comment:
It's a good practice to add `0s` as a delay time for animations. It doesn't change anything in your case, but it's a part of full animation syntax: `animation: name duration timing-function delay iteration-count direction fill-mode play-state;`. So your line should look like this: `animation: rotate 3600s steps(60) 0s infinite;`